Chaincase/BlazorApp.cs

```diff
@@ -33,19 +35,12 @@ public BlazorApp(IFileProvider fileProvider = null, Action<IServiceCollection> c
 					// Adds web-specific services such as NavigationManager
 					services.AddBlazorHybrid();
 					services.AddUIServices();
-					services.UseMicrosoftDependencyResolver();
-
-					// set Locator to reference this container, too
-					var resolver = Locator.CurrentMutable;
-					resolver.InitializeSplat();
-					resolver.InitializeReactiveUI();
-
-					resolver.RegisterLazySingleton(() => new IndexViewModel(), typeof(IndexViewModel));
-					resolver.RegisterLazySingleton(() => new SendViewModel(), typeof(SendViewModel));
```

Comment:
Why were these removed? There are pages dependent on these singletons.

Reply:
I think this gotta change b4 we merge

Reply:
`RegisterLazySingleton` is theoretically identical to `services.AddSingleton` as singletons in the microsoft DI are instantiated when used the first time.

I moved the registration of VMs to the UI project since they are UI services. See https://github.com/chaincase-app/chaincase/pull/97/files/246eb97607bacd98133d840ec90f04c172464e33#diff-8b0995535d9ad21942a6fb147e04d1bc51ae44f90e1324f4905469c3d0207259R21

Reply:
That's a welcome change. Thanks for the explanation